src/components/widgets/TableWidget/TableWidget.less

```diff
@@ -82,3 +82,14 @@
         }
     }
 }
+
+.showDotsMenu {
```

Comment:
We need a broader look for 'Customization' term. It is not enough to add a conditional flag for two different implementations, it is still hardcode and will lead to a bloating of TableWidget component and unintuitive public interface.

What you need is a `controlColumns` property
```tsx
interface ControlColumnProps extends ColumnProps<DataItem> {
    position: 'left' | 'right'
}
interface TableWidgetOwnProps {
    controlColumns?: ControlColumnProps[]
}
// ...
const controlColumnsLeft = props.controlColumns?.filer(item => item.position === 'left')
const controlColumnsRight =  props.controlColumns?.filer(item => item.position === 'right')
const columns = props.meta.fields.map // ...
const resultColumns = [ ...controlColumnsLeft, columns, ...controlColumnsRight ]
```
The default control column is extracted into a separate component:
```tsx
import React from 'react'
import styles  from './defaultControlColumn.less'

export function useDefaultControlColumn(props) {
    const dotColumn: ColumnProps<DataItem> = {
    // ...
    }
}
```
And from the client side:
```tsx
import {TableWidget as TeslerTable, useDefaultControlColumn} from '@tesler-ui/core'

const TableWidget: React.FC = (props) => {
    const controlColumn = useDefaultControlColumn(propsSubset) // do not pass all props, only required subset
    const controlColumns = React.useMemo(() => [controlColumn], [controlColumn])
    return <TeslerTable {...props} controlColumns={controlColumns} />
}
```
This way you will not be caught in situations when you have to further complicate TableWidgets logic due to user design system looks on control differntly: 

![tablePattern1](https://user-images.githubusercontent.com/13132449/82314403-70a0f500-99d2-11ea-864f-79919f94ba44.PNG)

![tablePattern2](https://user-images.githubusercontent.com/13132449/82314414-74cd1280-99d2-11ea-8110-0a407dffb5f5.PNG)